.vscode/cspell.json

```diff
@@ -408,6 +408,10 @@
       "filename": "sdk/storage/storage-blob/review/storage-blob.api.md",
       "words": ["Uncommited"]
     },
+    {
+      "filename": "sdk/storage/storage-file-share/review/storage-file-share.api.md",
+      "words": ["Sddl"]
```

Comment:
Sddl is known abbreviation for security-descriptor-definition-language:

https://learn.microsoft.com/en-us/windows/win32/secauthz/security-descriptor-definition-language